### ISSUE TITLE
Restructure contact section

### ISSUE THREAD

**Issue by Paktosan** [agdsn/sipa#362]
Since there are plenty of support requests that find their way to the executive board mailing list the contact form link for affiliates should be removed. Also the link to the regular contact form should be renamed to support form.
See Büro und Support 10.01.2018

**Comment by toothstone** [comment]
[Link to minutes.](https://www.wh2.tu-dresden.de/admin/documents/Protokolle/pdf/all/bus/Protokoll_2018-01-10_Public.pdf)

Simply removing the link is not really an option, since it was placed there so that partners (StuRa, university, Studentenwerk, delivieries...) have a way to get in touch.

Most publicly announced mailing lists (mitmachen@, vorstand@) receive misdirected support requests (without any contact form), I don't think there is much we can do about it. Also, it is usually less than 1 mail per month, so not really an unmanageable quantitiy.

However, the labels could be more clear. "Contact form for support request" or something like that would be easier to distinguish.

**Comment by sebschrader** [comment]
I agree with @toothstone and want to add, that we shouldn't blame the people who use the wrong address/form. Instead, we should enhance our UI. It's too difficult to find the correct support form. We're doing it wrong.

Look at [Selfnet's website](https://www.selfnet.de/). Instead of news, there are *BIG* buttons, that immediately lead the visitor to the places, were 95% of the visitors want to go. This place is not the news.

**Comment by toothstone** [comment]
We need to keep https://agdsn.de/sipa/pages/support/contacts up to date as well, since that is much more accessible on mobile devices.
